Resources/Prototypes/Entities/Objects/Misc/handy_flags.yml

```
@@ -0,0 +1,167 @@
+- type: entity
+  parent: BaseItem
+  id: BlankHandyFlag
+  name: blank handy flag
+  description: "Some piece of white cloth wound on a stick. Definitely not handy flag of France."
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/blank_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/blank_handy_flag.rsi
+
+- type: entity
+  parent: BaseItem
+  id: NTHandyFlag
+  name: Nanotrasen handy flag
+  description: "Glory to NT! Wait, they really made a handy flag for a corporation?"
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/NT_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/NT_handy_flag.rsi
+
+- type: entity
+  parent: BaseItem
+  id: SyndieHandyFlag
+  name: Syndicate handy flag
+  description: "For truly rebel patriots. Death to NT!"
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/syndie_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/syndie_handy_flag.rsi
+    
+- type: entity
+  parent: BaseItem
+  id: LGBTQHandyFlag
+  name: LGBTQ handy flag
+  description: "Be gay do crime handy flag"
```

```suggestion
  name: LGBTQ handy flag
  description: "The be gay do crime handy flag."
```

all flag descs here need periods at the end of them, the solid color flags mostly

also think this desc reads better like this, same for the wallmount flag